<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-GG0PK5N" failures="1" tests="10" name="Automation Tests" time="34.167" errors="0" timestamp="2024-08-29T18:41:43 EEST">
  <testcase classname="testcases.SauceDemo" name="checkItems" time="1.013"/>
  <testcase classname="testcases.SauceDemo" name="checkCart" time="0.318"/>
  <testcase classname="testcases.SauceDemo" name="checkRemoveButton" time="2.517"/>
  <testcase classname="testcases.SauceTestReset" name="resetTest" time="2.673"/>
  <testcase classname="testcases.SelectButtonTest" name="itemSorting" time="0.715"/>
  <testcase classname="testcases.UsersLoginTest" name="loginTest" time="0.264"/>
  <testcase classname="testcases.UsersLoginTest" name="loginTest" time="0.368"/>
  <testcase classname="testcases.UsersLoginTest" name="loginTest" time="0.261">
    <failure type="java.lang.AssertionError" message="Login attempt with username: locked_out_user and password: secret_sauce failed. expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: Login attempt with username: locked_out_user and password: secret_sauce failed. expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at testcases.UsersLoginTest.loginTest(UsersLoginTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="testcases.UsersLoginTest" name="loginTest" time="0.225"/>
  <testcase classname="testcases.UsersLoginTest" name="loginTest" time="0.253"/>
</testsuite> <!-- Automation Tests -->
